plugins/source/aws/resources/services/efs/filesystems.go

```diff
@@ -32,7 +32,12 @@ func Filesystems() *schema.Table {
 			{
 				Name:     "backup_policy_status",
 				Type:     arrow.BinaryTypes.String,
-				Resolver: ResolveEfsFilesystemBackupPolicyStatus,
+				Resolver: resolveEfsFilesystemBackupPolicyStatus,
+			},
+			{
+				Name:     "policy",
```

Comment:
```suggestion
				Name:     "file_system_policy",
```
so it is clear what is in the column, I think the name should be `file_system_policy`